### ISSUE TITLE
problem about session.run

### ISSUE THREAD

**Issue by lglhuada** [tensorflow/tensorflow#426]
Hi guys, I am very new to tensorflow. Now I am implementing a pairwise cnn model. While I got the error:

File "Train.py", line 72, in <module>
    train()
  File "Train.py", line 48, in train
    feature1,feature2,train_loss,_=sess.run([model.feature1,model.feature2,model.cost,model.train_op],feed)
  File "/usr/lib/python2.7/site-packages/tensorflow/python/client/session.py", line 345, in run
    results = self._do_run(target_list, unique_fetch_targets, feed_dict_string)
  File "/usr/lib/python2.7/site-packages/tensorflow/python/client/session.py", line 404, in _do_run
    target_list)
tensorflow.python.pywrap_tensorflow.StatusNotOK: Not found: FeedInputs: unable to find feed output Placeholder:0

Here in the sess.run function, feed is a dictionary containing input data and corresponding labels. feature1 and feature2 are the output of the same neural networks and finally I would like to compare feature1 and feature2.

Any help are more than welcome. Thanks!

**Comment by girving** [comment]
This is a better question for stackoverflow, since if we answer it there it'll be easy for future users to find the answer via search.  Can you repost and add the question link here?

**Comment by lglhuada** [comment]
hi girving, thanks for your advice. You can find the same question in stack flow with the link http://stackoverflow.com/questions/34150921/session-run-error-when-implement-a-pairwise-cnn-with-tensorflow

Look forward to your answer :0
